web/.env.example

```
@@ -20,3 +20,36 @@ BOT_API_SECRET=
+
+# Optional Sentry monitoring for the dashboard.
+# NEXT_PUBLIC_SENTRY_DSN is the browser-safe public DSN for client-side errors.
+NEXT_PUBLIC_SENTRY_DSN=
+
+# SENTRY_DSN enables server and edge runtime error capture.
+SENTRY_DSN=
+
+# Use matching environment names so browser/server events group together.
+# Examples: development, preview, staging, production
+SENTRY_ENVIRONMENT=development
+NEXT_PUBLIC_SENTRY_ENVIRONMENT=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SENTRY_ENVIRONMENT key should go before the SENTRY_ENVIRONMENT key

(UnorderedKey)